tests/tests/wgpu-validation/util.rs

```diff
@@ -39,3 +40,44 @@ fn staging_belt_random_test() {
         belt.recall();
     }
 }
+
+#[test]
+fn staging_belt_panics_with_invalid_buffer_usages() {
+    fn test_if_panics(usage: BufferUsages) -> bool {
+        std::panic::catch_unwind(|| {
+            let (device, _queue) = wgpu::Device::noop(&wgpu::DeviceDescriptor::default());
+            let _belt = wgpu::util::StagingBelt::new_with_buffer_usages(device.clone(), 512, usage);
+        })
+        .is_err()
```

Comment:
I would suggest checking at least a substring match of the panic message. In my experience, tests need to check for the *specific* error they are looking for, or they will start falsely passing in the future when changes accidentally make them trigger a different error.